src/drivers/drv_rgb_led.c

```diff
@@ -240,99 +98,114 @@ void rgb_send(int data) {
     rgb_dma_phase = 3;
 }
 
-// if dshot dma is used the routine is in that file
-#if !defined(USE_DSHOT_DMA_DRIVER) && defined(RGB_LED_DMA) && (RGB_LED_NUMBER > 0)
-
-void DMA1_Channel4_5_IRQHandler() {
-  DMA_Cmd(DMA1_Channel5, DISABLE);
-  DMA_Cmd(DMA1_Channel2, DISABLE);
-  DMA_Cmd(DMA1_Channel4, DISABLE);
-
-  TIM_DMACmd(TIM1, TIM_DMA_Update | TIM_DMA_CC4 | TIM_DMA_CC1, DISABLE);
-  DMA_ClearITPendingBit(DMA1_IT_TC4);
-  TIM_Cmd(TIM1, DISABLE);
-
-  rgb_dma_phase = 0;
-}
-#endif
-
-#else
-
-// sets all leds to a brightness
-void rgb_init() {
-  // spi port inits
-
-  if ((RGB_PIN == LL_GPIO_PIN_13 || RGB_PIN == LL_GPIO_PIN_14) && RGB_PORT == GPIOA) {
-    // programming port used
-
-    // wait until 2 seconds from powerup passed
-    while (time_micros() < 2e6)
-      ;
-  }
-
+void rgb_initIO(void)
+{
   LL_GPIO_InitTypeDef GPIO_InitStructure;
 
-  GPIO_InitStructure.Mode = LL_GPIO_MODE_OUTPUT;
+  // Enable Clock
+  LL_AHB1_GRP1_EnableClock(RGB_PIN_CLOCK);
+  // Config pin for digital output
+  GPIO_InitStructure.Mode = LL_GPIO_MODE_ALTERNATE;
   GPIO_InitStructure.OutputType = LL_GPIO_OUTPUT_PUSHPULL;
   GPIO_InitStructure.Pull = LL_GPIO_PULL_UP;
-  GPIO_InitStructure.Speed = LL_GPIO_SPEED_FREQ_HIGH;
+  GPIO_InitStructure.Speed = LL_GPIO_SPEED_FREQ_VERY_HIGH;
 
   GPIO_InitStructure.Pin = RGB_PIN;
   LL_GPIO_Init(RGB_PORT, &GPIO_InitStructure);
+  LL_GPIO_ResetOutputPin(RGB_PORT, RGB_PIN);
 
-  for (int i = 0; i < RGB_LED_NUMBER; i++) {
-    rgb_send(0);
-  }
+  // Bind IO pin to the alternate function
+  gpio_pin_init_af(&GPIO_InitStructure, RGB_PIN, RGB_TIM_AF);
```

Comment:
you are init-ing this pin twice. remove `LL_GPIO_Init` and `LL_GPIO_ResetOutputPin`
switch to using the helpers from drv_gpio exclusively,
see the target header comments for how to specify the pins correctly.